MPU/src/cascadiamc.cpp

```diff
@@ -87,10 +76,9 @@ uint16_t CASCADIAMC::getTorque()
 
 void CASCADIAMC::clearFault()
 {
-    int time = millis() + 250;
-    while(millis() < time) {
+    for(int tries = 0; tries < FAULT_CLEAR_ATTEMPTS; tries++)
+    {
```

Comment:
This will go infinitely fast, using up all fault clear attempts and realistically only sending one message

Reply:
Or Ig maybe not, but still kinda weird. Fault clear attempts should probably be spaced by at least 0.25 seconds